### PR TITLE
Add federalist pipeline

### PR DESCRIPTION
**Pull request by hursey013**
## Changes proposed in this pull request:
- Add federalist-pipeline.yml config for federalist-builder pipeline to deploy federalist-builder app to production

## security considerations
Needed for SCR

### CODE REVIEW

File: ci/federalist-pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+    - do: *test
+
+    on_failure:
+      put: src
+      resource: pr-main
+      params:
+        path: src
+        status: failure
+        base_context: concourse
+        context: test
```
Comment (apburnes) [inline ci/federalist-pipeline.yml:113]: We should update these `context` values to something like `test-federalist` so the pages production and the federalist production checks don't conflict with each other.

File: ci/federalist-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+  - name: slack
+    type: slack-notification
+    source:
+      url: ((slack-webhook-url))
+
+  - name: gh-status
+    type: cogito
+    check_every: 1h
+    source:
+      owner: 18F
```
Comment (apburnes) [inline ci/federalist-pipeline.yml:255]: I just realized we hardcoded this for all the pipelines so far. We may want to break this into variables to make transition of GH org simpler.

**Review by apburnes — COMMENTED** [review]
Looks good. Just a couple comments.

**Comment by hursey013** [comment]
@apburnes after much fiddling, opted to leave the repo owner/name as hardcoded in the federalist-pipeline, but did update the test context of both pipelines so they don't conflict

**Review by apburnes — APPROVED** [review]
:shipit:

**Pull request merged**